max_iterations: 5
iteration_contact_threshold: 0.025
